frontend/src/locales/en.yml

```diff
@@ -220,6 +220,9 @@ task:
     success: Success
     warn: Warning
     error: Error
+  earliest-allowed-time-no-modify: >-
+    You can not modify this field because you are not the assignee of this
+    issue.
```

Comment:
You can not modify this field because you are not the issue assignee or it's not pending to run.